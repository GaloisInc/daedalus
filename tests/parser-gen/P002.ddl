def Main = { $$ = Match1 '0' ; Match1 '0' }
