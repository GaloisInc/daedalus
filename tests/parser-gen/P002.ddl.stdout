module P002
 
--- Imports:
 
--- Type defs:
 
--- Rules:
 
P002.Main : Grammar (uint 8) =
  do ($$ : uint 8) <- Match {'0'}
     @Match {'0'}
     pure $$
 
P002.Main_ : Grammar {} =
  do @Match {'0'}
     @Match {'0'}
